exporters/user_events/src/metrics_exporter.cc

```diff
@@ -0,0 +1,109 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+#include "opentelemetry/exporters/user_events/metrics/exporter.h"
+
+#include "tracepoint/tracepoint.h"
+
+#include <memory>
+#include <mutex>
+
+OPENTELEMETRY_BEGIN_NAMESPACE
+
+namespace exporter {
+namespace user_events {
+namespace metrics {
+
+// -------------------------------- Constructors --------------------------------
+
+Exporter::Exporter() : Exporter(ExporterOptions()) {}
+
+Exporter::Exporter(const ExporterOptions &options)
+: options_(options),
+    aggregation_temporality_selector_{otlp_exporter::OtlpMetricUtils::ChooseTemporalitySelector(options_.aggregation_temporality)}
+{
+  int err;
+
+  err = tracepoint_open_provider(&provider_);
+
+  err = tracepoint_connect(&otlp_metrics_, &provider_, "otlp_metrics");
```

Comment:
Is "otlp_metrics" the name of tracepoint? In case of multiple exporter instances across applications, do all of them share the same name ?

Reply:
Thanks. Yes. I plan to read this from the config option in later PR. How do you think?